Add documents for backup, redeployment and observability (#21)
canonical/opencti-operator: docs/how-to/redeploy.md

> # How to Redeploy the OpenCTI Charm

Reviewer: ```suggestion
# How to redeploy the OpenCTI charm
```

Both the OpenCTI charm and the OpenCTI connector are stateless charms, so you 

> can redeploy the OpenCTI charm at any time.

Reviewer: Nitpick: Is there a command you can show here? If it's hard to generalize, then maybe something like the below suggestion (assuming that's even the correct command): 
```suggestion
can redeploy the OpenCTI charm at any time using `juju refresh`.
```

However, one thing to keep in mind is that the newly deployed charm must use the
same Juju application name as the previous OpenCTI charm. This is because the
OpenCTI charm uses the Juju application name as the OpenSearch index prefix. 

> Doing so ensures that the new OpenCTI charm inherits the old data created by

Reviewer: ```suggestion
Using the same Juju application name ensures that the new OpenCTI charm inherits the old data created by
```
(Nitpick; an attempt to be clearer)
